### ISSUE TITLE
Questions about the details of the  IXI  dataset.

### ISSUE THREAD

**Issue by Luke-Huang-Github** [icon-lab/SynDiff#31]
@onat-dalmaz Hi, thank you for your contribution. I have some questions about the details of the dataset.
   In IXI dataset, First of all, sequentially select the first 40 subjects from the dataset,with a (25,5,10) split for train, validation, and test sets. In paper "each subject 100 axial cross-sections with brain tissue were selected". Do you mean the data_train_contrast1.mat shape is (2500, width, height) -> (#images, width, height)?   #images = 25*100?

**Comment by muzafferozbey** [comment]
In our project, we used 25 subjects from the IXI dataset for training. As you said, approximately 100 axial cross-section images were extracted from each subject. Hence for our models, the shape of our data file is (#images, width, height). The number of training images was approximately 2500.  
However, this is not the case for the sample data on the GitHub page. We prefer not to repost datasets that were compiled and publicly shared by other entities to avoid potential conflicts. Therefore, the sample file contains only a few images.

**Comment by Luke-Huang-Github** [comment]
Hi. I have some questions about the details of the dataset.
Did the train dataset be registrated?
I train the IXI dataset (training set was not to be registrated), the result in my experiment (test set was be registrated) was very low, psnr = 17.33.

**Comment by dariusoneof** [comment]
I encountered the same issue where the PSNR during training on the IXI dataset is around 19. Could you please share how you resolved this problem? Thank you very much
